tests/CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ else()
     message("ENABLE_FULL_LIBCXX_TESTS not set - building some libcxx tests")
 endif()
 
+if(WIN32_SIMULATION)
```

Comment:
WIN32_SIMULATION will need to specified as an option in CMakeLists.txt in the root folder with default value as OFF as follows:
option(WIN32_SIMULATION "Windows Simulation Mode" OFF)

Reply:
Also "Add a space after if"

Reply:
Sure. I followed the coding style from the previous if.